### DISCUSSION TITLE
Could not open file version.json: No such file or directory

### DISCUSSION THREAD

**Discussion by RAlijev** [itzg/docker-minecraft-server#1968] [Q&A] [answered]
Hello Geoff and everyone,

I'm trying to host FTB Minecraft server in docker on my QNAP NAS, but getting the error (below) at the very beginning: 
![image](https://user-images.githubusercontent.com/123180799/220388579-e2be21d6-6812-46b3-ba14-23e5c0799529.png)

I'm quite new with Docker stuff, but with vanilla Minecraft server it works without issues.
If type is set to FTBA with two additional FTB properties for modpack version - it doesn't work...

Here are the settings I've used: 
![image](https://user-images.githubusercontent.com/123180799/220389983-baf43cc8-0752-4219-ba7c-67df3948811b.png)
Using GUI to create a container, due to some issues with Docker Compose and NAS I have.

Image used (latest at the time): 
![image](https://user-images.githubusercontent.com/123180799/220391374-7639d7a7-256f-465e-a40e-1496c7b59dcd.png)

Any ideas what is wrong?

P.S.: _eula.txt_ file is the only one created at the location.

**Comment by itzg** [comment, marked as answer]
It seems like it has gotten out of sync with what is downloaded. Since there's not yet a "force reinstall" option can you use `ls -a` to look for the dot/hidden files in your host path? Specifically there's `.ftb-installed` that if removed will force a reinstall.

**Reply by RAlijev** [reply]
Yes, you are right, there was a hidden `.ftb-installed`. Deleting it fixed the issue and server successfully installed. Thanks a lot Geoff! :)
